src/lib/swapper/api.ts

```diff
@@ -149,6 +151,19 @@ export enum SwapperName {
   OneInch = '1INCH',
 }
 
+export const ALL_SWAPPERS = [
```

Comment:
We don't seem to use this guy anywhere.

In any case it should probably be programmatic so we don't forget to update it:

```ts
export const ALL_SWAPPERS = Object.values(SwapperName).filter((value): value is SwapperName => 
  Object.values(SwapperName).includes(value) && value !== SwapperName.Test
);
```